tests/robustness/report/report.go

```diff
@@ -46,6 +47,12 @@ func testResultsDirectory(t *testing.T) string {
 	if err != nil {
 		t.Fatal(err)
 	}
+	suffix := fmt.Sprintf("%v", time.Now().UnixNano())
```

Comment:
Can you just change the `strings.ReplaceAll(t.Name(), "/", "_")` code on line 46?

Reply:
done